lib/genome/online_updater.rb

```diff
@@ -74,11 +74,15 @@ def create_interaction_claim_publication(interaction_claim, pmid)
     end
 
     def backfill_publication_information
-      DataModel::Publication.where(citation: nil).find_in_batches(batch_size: 10) do |publications|
+      DataModel::Publication.where(citation: nil).find_in_batches(batch_size: 100) do |publications|
         PMID.get_citations_from_publications(publications).each do |publication, citation|
           publication.citation = citation
           publication.save
         end
+        sleep(0.3)
+      end
+      DataModel::Publication.where(citation: "").each do |publication|
```

Comment:
Could a citation be nil here or only empty string?

Reply:
The loop above will fill all publication citations with either a real citation or an empty string (depending on what gets returned by the `citation` method in pmid.rb). If the citation is empty, the pmid actually doesn't exist in pubmed so we want to delete those publication records. I can't think of a circumstance under which a publication would still be nil after the loop above runs.

Reply:
Cool, sounds good!